.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contributing
+
+Everyone is welcome to contribute to this specification.
+
+Any simple editorial contribution can simply be done with a GitHub Pull Request.
+You can even do an inline edit of the file on GitHub.
+
+Note: Contributions to this repository are intended to become part of Recommendation-track documents governed by the
+[W3C Patent Policy](https://www.w3.org/Consortium/Patent-Policy-20040205/) and
+[Software and Document License](https://www.w3.org/Consortium/Legal/copyright-software). To make substantive contributions to specifications, you must either participate
+in the relevant W3C Working Group or make a non-member patent licensing commitment.
+
+## Style guide to contributors
+
+- the spec uses [ReSpec](https://www.w3.org/respec/)
+- the spec is tidied using [HTML5 Tidy](https://github.com/w3c/tidy-html5). For
+  instructions on running HTML5 tidy, see below.
+- Running tidy is optional. We run tidy on the server after every commit.
+- Put comments in front of sections, for better readability with
+  syntax coloring editors.
+
+## Commit message guidelines
+
+  - When relevant, include `closes #` and the issue number.
+  - The commit message should be short and concise.
+  - Prefix the commit message with one of the following:
+    - `Editorial:` -for editorial/non-normative change to the specification.
+    - `Chore:` - when fixing ReSpec stuff or other non-spec related stuff (e.g., CI).
+    - Normative changes should not be prefixed with anything. These go into the changelog.
+
+## Running HTML5 Tidy
+
+Please make sure you have [HTML5 Tidy](https://github.com/w3c/tidy-html5) installed.
```

Comment:
nit: that URI redirects to https://github.com/htacg/tidy-html5, not sure if it's worth updating or if you prefer living w/ the redirect

Reply:
Thinking with my "new contributor hat" and trying to figure out how to download and install tidy, the README in this repo isn't helpful. Building from source feels like overkill. Going to http://www.html-tidy.org/... it's a non-secure site, so I totally don't want to install the binaries there.

(edit: they _do_ have a secure version of the binaries page, at https://binaries.html-tidy.org/)

Grabbing the latest release from https://github.com/htacg/tidy-html5/releases is probably fine, or the site has links to package managers, e.g., `brew install tidy-html5` for macOS.

Perhaps we could add a sentence after `Please make sure you have [HTML5 Tidy](https://github.com/w3c/tidy-html5) installed.` like:

> Please make sure you have [HTML5 Tidy](https://github.com/w3c/tidy-html5) installed. You can download the [latest stable release from GitHub](https://github.com/htacg/tidy-html5/releases) or follow the "Get Tidy" instructions at http://www.html-tidy.org/.

Reply:
Yeah, brew instructions will do. Alternative, we just remove mention of it. GitHub actions handles it for random contributors.